pkg/rulefmt/rulefmt.go

```diff
@@ -45,19 +46,23 @@ type RuleGroups struct {
 	Groups []RuleGroup `yaml:"groups"`
 }
 
+type ruleGroups struct {
+	Groups []yaml.Node `yaml:"groups"`
+}
+
 // Validate validates all rules in the rule groups.
-func (g *RuleGroups) Validate() (errs []error) {
+func (g *RuleGroups) Validate(node ruleGroups) (errs []error) {
 	set := map[string]struct{}{}
 
-	for _, g := range g.Groups {
+	for j, g := range g.Groups {
 		if g.Name == "" {
-			errs = append(errs, errors.Errorf("Groupname should not be empty"))
+			errs = append(errs, errors.Errorf("Groupname should not be empty, line: %d, column: %d", node.Groups[j].Line, node.Groups[j].Column))
```

Comment:
Prometheus uses the `<line>:<col>: <Error message>` or  `<filename>:<line>:<col>: <Error message>` format for errors in the expression browser now, since that format is used by many compilers. Also, when using that format, VS Code allows to just click on the message to get the cursor to the error position.

Might make sense to do the same here.